src/components/DialogContentContainer/__stories__/DialogContentContainerExample.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import styles from "./DialogContentContainerExample.module.scss";
+
+const DialogContentContainerExample = () => {
+  return (
+    <div className={styles["content-example"]}>
```

Comment:
I think it may be confusing here to use an example component instead of show the div and p components (developers may think they should know more api's which are not in the docs)